mobile_config/tests/integrations/common/partial_migrator.rs

```diff
@@ -0,0 +1,108 @@
+use chrono::Utc;
+use sqlx::{
+    migrate::{Migration, Migrator},
+    PgPool,
+};
+use std::path::Path;
+
+pub struct PartialMigrator {
+    pool: PgPool,
+    versions: Vec<i64>,
+    path: String,
+}
+
+impl PartialMigrator {
+    pub async fn new(
+        pool: PgPool,
+        versions: Vec<i64>,
+        path: Option<String>,
```

Comment:
`PartialMigrator` is never used with migrations from a location other than default.
Can we drop the option? Or hold onto a the default constructed migrator?

```rs
pub struct PartialMigrator {
    pool: PgPool,
    versions: Vec<i64>,
    path: Migrator,
}
```

```rs
impl PartialMigrator {
    pub async fn new(pool: PgPool, versions: Vec<i64>) -> anyhow::Result<Self> {
        // ...

        Ok(Self {
            pool,
            versions,
            migrator: sqlx::migrate!(),
        })
    }
}
```